src/Polly.Core/Simmy/Outcomes/OutcomeStrategyOptions.TResult.cs

```diff
@@ -19,18 +21,6 @@ internal class OutcomeStrategyOptions<TResult> : MonkeyStrategyOptions
     /// <summary>
     /// Gets or sets the outcome generator to be injected for a given execution.
     /// </summary>
-    /// <remarks>
-    /// Defaults to <see langword="null"/>. Either <see cref="Outcome"/> or this property is required.
-    /// When this property is <see langword="null"/> the <see cref="Outcome"/> is used.
-    /// </remarks>
-    public Func<OutcomeGeneratorArguments, ValueTask<Outcome<TResult>?>>? OutcomeGenerator { get; set; }
-
-    /// <summary>
-    /// Gets or sets the outcome to be injected for a given execution.
-    /// </summary>
-    /// <remarks>
-    /// Defaults to <see langword="null"/>. Either <see cref="OutcomeGenerator"/> or this property is required.
-    /// When this property is <see langword="null"/> the <see cref="OutcomeGenerator"/> is used.
-    /// </remarks>
-    public Outcome<TResult>? Outcome { get; set; }
+    [Required]
+    public Func<OutcomeGeneratorArguments, ValueTask<Outcome<TResult>?>> OutcomeGenerator { get; set; }
```

Comment:
```suggestion
    public Func<OutcomeGeneratorArguments, ValueTask<Outcome<TResult>?>>? OutcomeGenerator { get; set; }
```

it's null bu default because we cannot assign it any sensible default value.

Reply:
It cannot be null since now we don't have the `Outcome` property just the `OutcomeGenerator`, see your previous comment https://github.com/App-vNext/Polly/pull/1682#discussion_r1349917952